### PR TITLE
[protobuf] Load C++,Java,Protobuf rules from Starlark

### PR DESCRIPTION
**Pull request by Yannic**
Cherry-pick of: protocolbuffers/protobuf#6432 and protocolbuffers/protobuf#6445

### CODE REVIEW

**Comment by Yannic** [comment]
Depends on https://github.com/bazelbuild/bazel/pull/9002

File: third_party/protobuf/3.6.1/BUILD [hunk 1]
```diff
@@ -24,9 +27,6 @@ config_setting(
 load(":protobuf.bzl", "py_proto_library")
 load(":compiler_config_setting.bzl", "create_compiler_config_setting")
 
-# Needed for #9006. Hopefully a future upstream version will include this line.
```
Comment (Yannic) [inline third_party/protobuf/3.6.1/BUILD]: I sent a PR to Protobuf to fix this, so I don't think this comment is needed.

Comment (lberki) [reply third_party/protobuf/3.6.1/BUILD]: Wow. You're awesome :)

**Comment by lberki** [comment]
Merged as deeeba3fc8003cbbcc73e04f14fc5b5275e72e5d so I'm closing this.

**Pull request closed without merging**